tests/main.nf.test.snap

```diff
@@ -0,0 +1,176 @@
+{
+    "Run with profile test": {
+        "content": [
+            23,
+            {
+                "BCFTOOLS_STATS": {
+                    "bcftools": 1.2
+                },
+                "BWAMEM1_MEM": {
+                    "bwa": "0.7.18-r1243-dirty",
+                    "samtools": 1.2
+                },
+                "FASTQC": {
+                    "fastqc": "0.12.1"
+                },
+                "GATK4_APPLYBQSR": {
+                    "gatk4": "4.5.0.0"
+                },
+                "GATK4_BASERECALIBRATOR": {
+                    "gatk4": "4.5.0.0"
+                },
+                "GATK4_MARKDUPLICATES": {
+                    "gatk4": "4.5.0.0",
+                    "samtools": "1.19.2"
+                },
+                "INDEX_CRAM": {
+                    "samtools": 1.2
+                },
+                "MOSDEPTH": {
+                    "mosdepth": "0.3.8"
+                },
+                "SAMTOOLS_STATS": {
+                    "samtools": 1.2
+                },
+                "STRELKA_SINGLE": {
+                    "strelka": "2.9.10"
+                },
+                "VCFTOOLS_TSTV_COUNT": {
+                    "vcftools": "0.1.16"
+                },
+                "Workflow": {
+                    "nf-core/sarek": "v3.5.0dev"
+                }
+            },
+            [
+                "csv",
+                "markduplicates.csv",
+                "markduplicates_no_table.csv",
+                "recalibrated.csv",
+                "variantcalled.csv",
```

Comment:
why are these not stable?

Reply:
because they contain the full path and not the relative one, I need to sort this out